Description: Upstream changes introduced in version 1.4.91-0oneiric
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 ros-groovy-octomap (1.4.91-0oneiric) oneiric; urgency=high
 .
   * autogenerated
 .
 The person named in the Author field signed this changelog entry.
Author: Armin Hornung <hornunga@informatik.uni-freiburg.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ ros-groovy-octomap-1.4.91/bloom.conf
@@ -0,0 +1,3 @@
+[catkin]
+	upstream = https://octomap.svn.sourceforge.net/svnroot/octomap/tags/v1.4.2/octomap
+	upstreamtype = svn
